chart/templates/ws-proxy-role.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app: {{ template "gitpod.fullname" . }}
+    component: ws-proxy
+    kind: role
+    stage: {{ .Values.installation.stage }}
+  name: ws-proxy
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - update
```

Comment:
nit: does ws-proxy really need update and patch?

Reply:
Good point. No. Removed.